metaproc-connector/src/com/axonivy/connector/metaproc/auth/AuthTokenFeature.java

```diff
@@ -52,7 +53,10 @@ public void filter(ClientRequestContext context) throws IOException {
               .accept(MediaType.APPLICATION_JSON).post(entity)) {
         var node = response.readEntity(ObjectNode.class);
         TOKEN = node.get("loginResult").get("token").asText();
+        TOKEN_EXPIRED_TIME = System.currentTimeMillis() + Long.valueOf(Ivy.var().get("metaproc.TokenExpiredDuration"));
         context.getHeaders().add(TOKEN_HEADER, TOKEN);
+      } catch (Exception e) {
+        Ivy.log().error(e.getMessage());
```

Comment:
with this log, the stacktrace is swallowed ... with no trace what lead us here. 
also it's common sense in productive java code to enrich the message being logged with the local context